### ISSUE TITLE
Number of 18mm Plywood Sheets Needed

### ISSUE THREAD

**Issue by EJN1** [space10-community/the-growroom#9]
The Manual specifies 13 sheets of 18mm plywood but the .dxf file only seems to show 12. What am I missing - Thanks

**Comment by silicakes** [comment]
IMO, it seems like you'll be cutting several parts from each sheet - so it consolidates into less than 13.

**Comment by cwiegman** [comment]
I started actually making the Growroom and before heading out to purchase the remaining 7 boards, I noticed this 12 v 13 difference too. So I made a count of parts that will be made from the dxf vs the milled diagram that we should have, and I could be wrong but I think 13 boards is the correct number. I will let you but I notice only one of the F'circle parts missing, so only half a board is needed. My plan is to actually change some of the nesting with ~80 wedges, placing some on the scrap areas of other boards, copy a 5th F into that freedup space and and only use 12 boards.

**Comment by HusumLindholm** [comment]
Hi all,

It is true that you only need 12 sheets of 18mm plywood.
Regarding the missing F piece, the drawings have been updated and include all pieces - still on 12 sheets. Further more we have cleaned up the lines, so the files now consist of arcs and lines only, with much fewer control points. Furthermore we have uploadet a package that include each sheet in single files with numbers.

Best regards
Husum&Lindholm
